### DISCUSSION TITLE
What are transparent shadows?

### DISCUSSION THREAD

**Discussion by jwwalker** [google/filament#4340] [Q&A] [answered]
I see that transparent shadows are now supported, but I'm having trouble figuring out what that means.  I don't see how it makes sense for a shadow to be literally transparent, since a shadow is not a physical object.  Shadows cast onto transparent objects?  Shadows cast by transparent objects?  Sorry if that's a dumb question.

**Comment by bejado** [comment, marked as answer]
Transparent shadows are shadows cast by a transparent object.

**Reply by romainguy** [reply]
And the shadow is transparent in the sense that it doesn't block 100% of the incoming light.

**Reply by jwwalker** [reply]
Thanks.  And I see now that it's explained more fully in the `transparentShadow` discussion in the materials guide.
